src/Pages/Profile/Styles/userProfile/UserProfile.styles.ts

```diff
@@ -0,0 +1,29 @@
+import styled from "styled-components"
+
+export const UserProfileSection = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 4rem 0 2rem;
+  padding: 4rem 0 0;
+  border-bottom: 1px solid;
+`
+export const UserProfileImageContainer = styled.div`
+  height: 16rem;
+  width: 16rem;
+  border-radius: 50%;
```

Comment:
전체적인 통일성을 위해
```suggestion
  border-radius: ${({ theme }) => theme.radius.circle};
```
로 바꿔보시면 좋을 것 같습니다!!😊